### PR TITLE
add more to Maths

### PR DESCRIPTION
**Pull request by shellhub**

### CODE REVIEW

File: Maths/FindMin.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        System.out.println("min = " + findMax(array));
+    }
+
+    /**
+     * find min of array
+     *
+     * @param array the array contains element
+     * @return min value
+     */
+    public static int findMax(int array[]) {
```
Comment (yanglbme) [inline Maths/FindMin.java]:
```suggestion
    public static int findMax(int[] array) {
```

File: Maths/FindMax.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        System.out.println("max = " + findMax(array));
+    }
+
+    /**
+     * find max of array
+     *
+     * @param array the array contains element
+     * @return max value
+     */
+    public static int findMax(int array[]) {
```
Comment (yanglbme) [inline Maths/FindMax.java]:
```suggestion
    public static int findMax(int[] array) {
```

Comment (shellhub) [reply Maths/FindMax.java]: Thanks very much!!! Please review again.

File: Maths/FindMin.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package Maths;
+
+public class FindMin {
+
+    //Driver
+    public static void main(String[] args) {
+        int array[] = {2, 4, 9, 7, 19, 94, 5};
```
Comment (yanglbme) [inline Maths/FindMin.java]:
```suggestion
        int[] array = {2, 4, 9, 7, 19, 94, 5};
```

File: Maths/FindMax.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package Maths;
+
+public class FindMax {
+
+    //Driver
+    public static void main(String[] args) {
+        int array[] = {2, 4, 9, 7, 19, 94, 5};
```
Comment (yanglbme) [inline Maths/FindMax.java]:
```suggestion
        int[] array = {2, 4, 9, 7, 19, 94, 5};
```

Comment (shellhub) [reply Maths/FindMax.java]: @yanglbme review again

**Pull request merged**